src/components/ProblemsTable.tsx

```diff
@@ -244,6 +283,27 @@ export const ProblemsTable = ({
 
   return (
     <Table
+      onRowClick={(event: React.MouseEvent<Element, MouseEvent>, row: Row<TableRow | TableRow['subRows']>) => {
+        const element = event.target as HTMLTableRowElement;
```

Comment:
move the whole onRowClick function to separate function as writing everything in line decreases the readability